src/location-registry/config.py

```diff
@@ -4,6 +4,8 @@
 import os
 import sys
 from dotenv import load_dotenv
+import ee
```

Comment:
simply curious what this ee package does. could you throw some light @danielogen

Reply:
Google Earth Engine package. It's used to get altitude, terrain, population density etc. of an area

Reply:
@ja-odur see @lilly-muyama above. thanks